templates/takeovers/_iot-device-management-whitepaper.html

```diff
@@ -0,0 +1,16 @@
+{% with title="Secure IoT device management",
+subtitle="Build and deploy a central IoT management solution",
+class="p-takeover--dark",
+header_image="https://assets.ubuntu.com/v1/9c1315fb-IOT_Ubuntu_devices_inforgrapic+v3.svg",
+image_style="",
+image_height="300",
+image_width="500",
+image_hide_small=true,
+equal_cols=true,
+primary_url="/engage/iot-device-management-whitepaper",
```

Comment:
This should have the utm query parameter: `?utm_source=Takeover&utm_medium=Takeover&utm_campaign=CY19_IOT_Ubuntu Core_Whitepaper_IOTDeviceMangement`